Store/DAL/Interfaces/IRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using DAL.Entities;
+
+namespace DAL.Interfaces
+{
+    public interface IRepository
+    {
+        public IEnumerable<Product> GetProducts();
```

Comment:
Get rid of 'Products' postfix, when the interface will be named correctly - we will understand what we get